docs/_library/affective-conflict-resolution-task.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: experiment
+name:  "affective-conflict-resolution-task"
+maintainer: "@JuneOh710"
+github: "https://github.com/JuneOh710/affective-conflict-resolution-task"
+preview: "https://juneoh710.github.io/affective-conflict-resolution-task/"
+tags:
+- test
+- experiment
+---
+This was a legacy experiment that has been ported into its Experiment Factory Reproducible Container version. If you'd like to make the experiment, it's documentation, or use better, please contribute at the repositories
```

Comment:
Do you want to change the descriptions this time to better reflect the experiments? I don't think it's correct that this was a legacy expfactory experiment.